src/Validator.php

```diff
@@ -60,6 +60,21 @@ function ($value) {
     }
 
     /**
+     * Assert that each specified variable is a number.
+     *
+     * @return \Dotenv\Validator
+     */
+    public function isInteger()
+    {
+        return $this->assertCallback(
+            function ($value) {
+                return (ctype_digit($value) == true);
```

Comment:
The `ctype_digit` function already returns a boolean. Change this to `return ctype_digit($value)`.